listener/listener.go

```diff
@@ -267,6 +268,30 @@ func (l *Listener) Close(ctx context.Context) error {
 	return nil
 }
 
+// GetActiveMessageCount gets the active message count of a topic subscription
+func (l *Listener) GetActiveMessageCount(ctx context.Context, topicName, subscriptionName string) (int32, error) {
+	topicEntity, err := getTopicEntity(ctx, topicName, l.namespace)
```

Comment:
Since we do getTopicEnitty and getSubscription Entity is it two outgoing calls to arm? 
Asking since it will impact our thorttling if we hit arm twice. 

Can we reuse the topicentity from creation in getSubscriptionEntity or does it need to be a freshly pulled one?

Reply:
Yes, we don't really need a real time topic entity. Made the changes.